metals/src/main/scala/scala/meta/internal/metals/ClientCommands.scala

```diff
@@ -170,6 +171,51 @@ object ClientCommands {
     """.stripMargin
   )
 
+  val UpdateTestExplorer = new ListParametrizedCommand[BuildTargetUpdate](
+    "metals-update-test-explorer",
+    "Update Test Explorer",
+    "Notifies the client that the test explorer model has to be updated",
+    """
+      |Sends to the client an Array of `BuildTargetUpdate`s.
```

Comment:
Should it maybe be an object? It's easier to modify it later on.

Reply:
>It's easier to modify it later on.

I'm curious, easier how?

Reply:
You can just add an additional parameter when needed, which will be impossible in an array. Though that might never be needed.